ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -48,23 +79,66 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
         <style>
           :host {
             @apply --layout-vertical;
+            transition: all 0.7s;
           }
           paper-checkbox {
-            --paper-checkbox-label-spacing: 10px;
-            margin-bottom: 5px;
+            width: 100%;
+            margin-top: 6px;
+            --paper-checkbox-label-spacing: 8px;
+          }
+          nuxeo-collapsible {
+            --nuxeo-collapsible-heading-padding: 0;
+          }
+          label {
+            @apply --nuxeo-label;
+          }
+          .showButton a {
+            display: block;
+            margin-top: 4px;
           }
         </style>
 
-        <dom-repeat items="{{buckets}}">
+        <dom-if if="[[_isEmpty]]">
           <template>
-            <paper-checkbox noink checked="{{item.checked}}" on-change="_computeValues">
-              [[item.label]] ([[item.docCount]])
-            </paper-checkbox>
+            <label>[[label]]</label>
+            <span>[[i18n('checkboxAggregation.noResults')]]</span>
           </template>
-        </dom-repeat>
-        <dom-if if="[[_isEmpty]]">
+        </dom-if>
+
+        <dom-if if="[[!_isEmpty]]">
           <template>
-            <label>[[i18n('checkboxAggregation.noResults')]]</label>
+            <dom-if if="[[!collapsible]]">
+              <template>
+                <label>[[label]]</label>
+                <dom-repeat items="{{buckets}}">
+                  <template>
+                    <paper-checkbox noink checked="{{item.checked}}" on-change="_computeValues">
+                      [[item.label]] ([[item.docCount]])
+                    </paper-checkbox>
+                  </template>
+                </dom-repeat>
+              </template>
+            </dom-if>
+
+            <dom-if if="[[collapsible]]">
+              <template>
+                <nuxeo-collapsible>
+                  <label slot="heading">[[label]]</label>
+                  <dom-repeat items="{{_visibleBuckets}}">
+                    <template>
+                      <paper-checkbox noink checked="{{item.checked}}" on-change="_computeValues">
+                        [[item.label]] ([[item.docCount]])
+                      </paper-checkbox>
+                    </template>
+                  </dom-repeat>
+                  <span hidden$="[[_hideShowMore(buckets)]]" class="showButton">
```

Comment:
visibleItems is also a dependency of this method:
```suggestion
                  <span hidden$="[[_hideShowMore(buckets, visibleItems)]]" class="showButton">
```
Without it, the show more won't show up if, for example, someone changes the visible items to a smaller number dynamically.